Server/appsettings.Development.json

```diff
@@ -7,7 +7,7 @@
     }
   },
   "ConnectionStrings": {
-    "Database": "Server=127.0.0.1;Port=5432;Database=postgres;User Id=postgres;Password=postgres"
+    "Database": "Server=127.0.0.1;Port=5432;Database=postgres;User Id=postgres;Password=0000"
```

Comment:
These are the parameters used by the docker container, you shouldn't change them